eng/pipelines/typespec-validation-all.yml

```diff
@@ -4,7 +4,19 @@ trigger:
     - main
     - typespec-next
 
-pr: none
+pr:
+  branches:
+    include:
+    - main
+    - typespec-next
+  paths:
+    include:
+    - .gitattributes
+    - .prettierrc.json
```

Comment:
I don't think we'll need to trigger on prettier settings?

Reply:
Actually we do, because `tsv` runs `tsp format` which uses `prettier` which depends on `.prettierrc.json`.